core/src/main/java/com/google/cloud/sql/core/InternalConnectorRegistry.java

```diff
@@ -261,4 +267,55 @@ private Connector createConnector(ConnectionConfig config) {
         refreshTimeoutMs,
         serverProxyPort);
   }
+
+  private ConnectionConfig resolveConfig(ConnectionConfig config) {
+    if (config.getNamedConnection() != null) {
+      ConnectorKey namedConfig = resolveKey(config);
+      if (namedConfig == null) {
+        throw new IllegalArgumentException(
+            "Named configuration " + config.getNamedConnection() + " does not exist.");
```

Comment:
This message is not clear. Maybe "Named connection "%" does not exist."

Reply:
Fixed.